core/domain/suggestion_jobs_one_off.py

```diff
@@ -236,3 +238,122 @@ def map(item):
+    KEY_DELIMITER = '.'
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [
+            suggestion_models.GeneralSuggestionModel,
+            user_models.UserContributionRightsModel
+        ]
+
+    @staticmethod
+    def map(item):
+        if item.deleted:
+            return
+
+        if isinstance(item, suggestion_models.GeneralSuggestionModel):
+            # Exit early if the suggestion type is not a part of the
+            # Contributor Dashboard or if the suggestion is not currently in
+            # review.
+            if item.suggestion_type == (
+                    suggestion_models.SUGGESTION_TYPE_EDIT_STATE_CONTENT) or (
+                        item.status != suggestion_models.STATUS_IN_REVIEW):
+                return
+            suggestion = suggestion_services.get_suggestion_from_model(item)
+            yield ('%s%s%s%s%s' % (
+                (
+                    PopulateCommunityContributionStatsOneOffJob
+                    .ITEM_TYPE_SUGGESTION
+                ), PopulateCommunityContributionStatsOneOffJob.KEY_DELIMITER,
+                suggestion.suggestion_type,
+                PopulateCommunityContributionStatsOneOffJob.KEY_DELIMITER,
+                suggestion.language_code), 1)
+
+        else:
+            if item.can_review_translation_for_language_codes:
+                for language_code in (
+                        item.can_review_translation_for_language_codes):
+                    yield ('%s%s%s%s%s' % (
+                        (
+                            PopulateCommunityContributionStatsOneOffJob
+                            .ITEM_TYPE_REVIEWER
+                        ),
+                        (
+                            PopulateCommunityContributionStatsOneOffJob
+                            .KEY_DELIMITER
+                        ),
+                        (
+                            PopulateCommunityContributionStatsOneOffJob
+                            .ITEM_CATEGORY_TRANSLATION
+                        ),
+                        (
+                            PopulateCommunityContributionStatsOneOffJob
+                            .KEY_DELIMITER
+                        ),
+                        language_code), 1)
+            if item.can_review_questions:
+                yield ('%s%s%s%s%s' % (
+                    (
+                        PopulateCommunityContributionStatsOneOffJob
+                        .ITEM_TYPE_REVIEWER
+                    ),
+                    PopulateCommunityContributionStatsOneOffJob.KEY_DELIMITER,
+                    (
+                        PopulateCommunityContributionStatsOneOffJob
+                        .ITEM_CATEGORY_QUESTION
+                    ),
+                    PopulateCommunityContributionStatsOneOffJob.KEY_DELIMITER,
+                    constants.DEFAULT_LANGUAGE_CODE), 1)
+
+    @staticmethod
+    def reduce(key, values):
+        community_contribution_stats_model = (
```

Comment:
Just call this `stats_model`, since some of the names are getting too long.

Reply:
Made the change!